parser.y

```diff
@@ -1595,6 +1595,36 @@ AlterTableSpec:
 		yylex.AppendError(yylex.Errorf("The WITH/WITHOUT VALIDATION clause is parsed but ignored by all storage engines."))
 		parser.lastErrorAsWarn()
 	}
+| "ALTER" "CHECK" Identifier EnforcedOrNot
```

Comment:
These indentions are inconsistent here. Could you fix it?

Reply:
> These indentions are inconsistent here. Could you fix it?

ok